bridge-circuit-host/src/bridge_circuit_host.rs

```diff
@@ -175,7 +175,7 @@ pub fn prove_bridge_circuit(
     let circuit_g16_proof = CircuitGroth16Proof::from_seal(risc0_g16_256);
     let ark_groth16_proof: ark_groth16::Proof<Bn254> = circuit_g16_proof.into();
 
-    tracing::warn!(
+    tracing::info!(
```

Comment:
Can we make these tracing::debug please? Somebody using the bridge is not going to be interested in seeing this information as this is debug info, meant for us to see.